docs/user/user/metrics/event.md

```diff
@@ -1,5 +1,7 @@
 # Events
 
+> **Important:** events are the most expensive metric type to record, transmit, store and analyze, so they should be used sparingly, and only when none of the other metric types are sufficient for answering your question.
+
```

Comment:
nit: IMO this should come after the initial description at the end of this section, not at the start.

Reply:
Makes sense, I made the adjustment.